react/features/notes/components/NotesDialog.web.js

```diff
@@ -123,6 +126,29 @@ class NotesDialog extends Component<Props, State> {
         );
     }
 
+    /**
+     * Renders the download file button.
+     *
+     * @inheritdoc
+     */
+    _renderDownloadFileButton() {
+
+        const {
+            t /* The following fixes a flow error: */ = _.identity
+        } = this.props;
+
+        return (
+            <Button
+                appearance = 'primary'
```

Comment:
We have both buttons (copy text and download text) as appearance = 'primary'. Probably is a good idea to have one of them more dominant as per styling contrast, perhaps download is more important as copy text. I'd suggest to change it so.

Reply:
all fixes about UI I will include in the next pr